libnetdata/log/log.c

```diff
@@ -822,7 +822,8 @@ void fatal_int( const char *file, const char *function, const unsigned long line
     char action_data[70+1];
 	snprintfz(action_data, 70, "%04lu@%-10.10s:%-15.15s/%d", line, file, function, __errno);
 	char action_result[60+1];
-	snprintfz(action_result, 60, "%s:%s",program_name, strcmp(program_name,"STREAM_RECEIVER")?netdata_thread_tag():"[x]");
+
```

Comment:
This PR is a good opportunity to remove those tab characters here and convert them to spaces.

Reply:
Done! Thank you observe this!

Reply:
The comparison got lost though.

Reply:
no, I'm wrong, disregard.

Reply:
All right!